docs/source/docs.rst

```diff
@@ -75,14 +92,14 @@ Exceptions
 .. autoexception:: cognite.extractorutils.configtools.InvalidConfigError
 
 
-``throttle`` - Tools for throttling
------------------------------------
+.. ``throttle`` - Tools for throttling
+.. -----------------------------------
 
-.. automodule:: cognite.extractorutils.throttle
-    :members:
-    :undoc-members:
-    :inherited-members:
-    :show-inheritance:
+.. .. automodule:: cognite.extractorutils.throttle
+..     :members:
+..     :undoc-members:
+..     :inherited-members:
+..     :show-inheritance:
```

Comment:
I think we refactored this module out in a previous release, could you double check that and remove it if we did?

Reply:
It was removed, I left there just so you can see it and maybe tell me something that I don't know. I also left the `cognite.extractorutils.uploader_extractor`, commented, just so we know that there is something there but it's not documented, do you want me to remove?

Reply:
I think we should just remove it